nise/__init__.py

```diff
@@ -1,3 +1,3 @@
-__version__ = "4.5.7"
+__version__ = "4.5.8"
```

Comment:
Since the interface is changing and we are removing command line options, this should be at leaste be a minor version update. This is more than patch.

Reply:
Suggestion accepted.